sample-compose/app/build.gradle.kts

```diff
@@ -116,60 +119,49 @@ kapt {
 }
 
 dependencies {
-    implementation(project(Module.DATA))
-    implementation(project(Module.DOMAIN))
+    implementation(projects.data)
+    implementation(projects.domain)
 
     implementation(fileTree(mapOf("dir" to "libs", "include" to listOf("*.jar"))))
 
-    implementation("androidx.core:core-ktx:${Versions.ANDROIDX_CORE_KTX_VERSION}")
-    implementation("androidx.lifecycle:lifecycle-runtime-ktx:${Versions.ANDROIDX_LIFECYCLE_VERSION}")
-    implementation("androidx.lifecycle:lifecycle-runtime-compose:${Versions.ANDROIDX_LIFECYCLE_VERSION}")
+    implementation(libs.bundles.androidx)
 
-    implementation(platform("androidx.compose:compose-bom:${Versions.COMPOSE_BOM_VERSION}"))
-    implementation("androidx.compose.ui:ui")
-    implementation("androidx.compose.ui:ui-tooling")
```

Comment:
I don't see `ui-tooling` import with debug (please fix) after migrating 💭

Reply:
Updated in [fb7d65c](https://github.com/nimblehq/android-templates/pull/596/commits/fb7d65c372c9e1467867c8db1770ed20515517ee)